### DISCUSSION TITLE
Trino 410 + Superset  2.0.0  not displaying tables in UI

### DISCUSSION THREAD

**Discussion by rnpaiva** [trinodb/trino#17398] [Q&A]
# Steps 
- Install trino 410 connector using helm chart and configure as example  : 
''' 
connectors:
  athena.properties: |
    connector.name=hive
    hive.hdfs.impersonation.enabled=true
    hive.metastore=glue
    hive.metastore.glue.region=us-east-1
    hive.metastore.glue.aws-access-key=${ENV:SECRET_AWS_ACCESS_KEY_ID}
    hive.metastore.glue.aws-secret-key=${ENV:SECRET_AWS_SECRET_ACCESS_KEY}
    hive.s3.aws-access-key=${ENV:SECRET_AWS_ACCESS_KEY_ID}
    hive.s3.aws-secret-key=${ENV:SECRET_AWS_SECRET_ACCESS_KEY}
'''
- Install superset 2.0.0 
- Configure trino in superset SQLALCHEMY URI :  trino://admin@trino.trino.svc.cluster.local:8080/athena

# Expected 
- See all tables in a schemas using trino 410 connector
- Hint: version 391 it's working , but I need to upgrade trino due other issues fixed  

# Behavior
- Not showing a single table in schema  ( see atached) 

![Screenshot from 2023-03-14 11-10-40](https://user-images.githubusercontent.com/97540715/225028168-3ceccc81-205d-4060-b3bf-ec2dfeafa236.png)

**Comment by hovaesco** [comment]
Hi, do you have logs from Trino? Can you query the mentioned catalog in Trino 410 from other client tool like trino-cli?

**Comment by mdesmet** [comment]
What version of trino-python-client is installed?
